Client/src/components/Shared/NotLoggedInView/NotLoggedInView.jsx

```
@@ -0,0 +1,81 @@
+    },
+    {
+      icon: <Activity size={36} />,
+      title: "Performance Insights",
+      description: "Optimize your ads based on real emotional engagement data"
+    }
+  ];
+
+  return (
+    <div className="h-screen w-full flex flex-col items-center justify-center relative overflow-hidden">
+      {/* Background elements */}
+      <div className="absolute top-10 -left-40 w-96 h-96 bg-blue-700 opacity-20 rounded-full filter blur-3xl"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)